molgenis-annotators-cmd/src/main/java/org/molgenis/annotation/cmd/CmdLineAnnotator.java

```diff
@@ -1,4 +1,4 @@
-package org.molgenis.data.annotation.cmd;
+package org.molgenis.annotation.cmd;
```

Comment:
this is fixed, however the line note is not on the line of the problem, so github keeps showing this comment.